devito/passes/clusters/aliases.py

```diff
@@ -97,19 +105,21 @@ def cire(cluster, mode, sregistry, options, platform):
             break
 
         # Search aliasing expressions
-        aliases = collect(extracted, min_storage, ignore_collected)
+        aliases = collect(extracted, ignore_collected, options)
 
         # Rule out aliasing expressions with a bad flops/memory trade-off
         chosen, others = choose(exprs, aliases, selector)
         if not chosen:
             # Do not waste time
             continue
 
-        # Create Aliases and assign them to Clusters
-        clusters, subs = process(cluster, chosen, aliases, max_par, sregistry, platform)
+        # AliasMapper -> Schedule -> [Clusters]
```

Comment:
Maybe fuse with old comment?

Reply:
I think this one is fine. What's troubling you? basically it's describing the lowering from AliasMapper to a list of Clusters

Reply:
hmm.. I felt like the verbs Create and assign to Clusters was better...but np..